test/e2e/snaps/test-snap-cronjob.spec.js

```diff
@@ -87,10 +76,12 @@ describe('Test Snap Cronjob', function () {
         });
 
         // try to click on the Ok button and pass test if window closes
-        await driver.clickElementAndWaitForWindowToClose({
+        await driver.clickElementSafe({
           text: 'OK',
           tag: 'button',
         });
+
+        await driver.waitForWindowToClose(dialogHandle);
```

Comment:
we verify that the MetaMask dialog was closed, either by ourselves or automatically